api/list/list_api_test.go

```diff
@@ -0,0 +1,105 @@
+package list
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"gotest.tools/assert"
+
+	"github.com/gojekfarm/albatross/pkg/logger"
```

Comment:
as we discussed albatross-cli could be in `pkg` and not the ones needed for albatross.

Reply:
This is also used for passing a logger to the helm library. Since it was used inside helmcli, kept the logger under pkg.